src/app/inventory/DraggableInventoryItem.tsx

```diff
@@ -13,6 +13,11 @@ interface Props {
 let dragTimeout: number | null = null;
 
 export default function DraggableInventoryItem({ children, item }: Props) {
+  function canDrag() {
+    return (!item.location.inPostmaster || item.destinyVersion === 2) && item.notransfer
+      ? item.equipment
+      : item.equipment || item.bucket.hasTransferDestination;
+  }
   const [_collected, dragRef] = useDrag<DimItem>(
```

Comment:
I think you can use this "collected" return value to map things like this out of the useDrag function: https://react-dnd.github.io/react-dnd/docs/overview#connectors